src/test/java/seedu/taskell/logic/LogicManagerTest.java

```diff
@@ -156,23 +156,19 @@ public void execute_add_invalidArgsFormat() throws Exception {
         assertCommandBehavior(
                 "add wrong args wrong args", expectedMessage);
         assertCommandBehavior(
-                "add Valid Name 12345 e/valid@email.butNoPhonePrefix a/valid, address", expectedMessage);
+                "add Valid Name valid@email.butNoPrefix a/valid, address", expectedMessage);
```

Comment:
Did u accidentally remove e/? Meaning "e/valid@email.butNoPrefix" and not "valid@email.butNoPrefix"